### PR TITLE
Cc 53 create tests

### PR DESCRIPTION
**Pull request by falcaodiogo**
Plzzzzzzzzzzzzzzzzzzz

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <name>backend</name>
   <description>backend</description>
   <properties>
-    <java.version>21</java.version>
+    <java.version>17</java.version>
```
Comment (RGarrido03) [inline pom.xml:17]: AI AI AI AI AI

**Review by RGarrido03 — CHANGES_REQUESTED** [review]
THE MF JAVA VERSION YOU DAMN B*TCH

File: pom.xml (hunk 1 above)
Comment (falcaodiogo) [reply pom.xml]: image

**Review by RGarrido03 — APPROVED** [review]
You used to deserve some rest. But then you changed the Java version.

**Pull request merged**